README.md

```diff
@@ -8,6 +8,15 @@ PyMC3 educational resources, including the PyMC3 port of the following books (or
 -  [PyMC3 port of the book "Bayesian Statistical Methods" by Brian J. Reich and Sujit K. Ghosh](https://github.com/pymc-devs/resources/tree/main/BSM)
 -  [PyMC3 port of the book "Bayesian Data Analysis" by Andrew Gelman, John B. Carlin, Hal S. Stern, David B. Dunson, Aki Vehtari and Donald B. Rubin](https://github.com/pymc-devs/resources/tree/main/BDA3)
 
+## How to contribute
+Thanks wanting to contribute! These resources are a community effort and we, and all future resource users, appreciate your help.
+
+If just starting
+1. Reading the [contributing guide for pymc](https://github.com/pymc-devs/pymc/blob/main/CONTRIBUTING.md) is a good place to start. The guide will familiarize you with the high level tools and workflow.
```

Comment:
I would use https://docs.pymc.io/en/latest/contributing/index.html instead. The github one should be kept because it is the one that gets linked automatically everywhere on github, but we can write much better guides in the documentation and reduce scatter of content (if contributing points to doc, wiki points to docs... everyone will go to the docs to find whatever they search for, if we do the other way around, from docs point to wiki, from docs point to github... it will have the opposite effect)

Reply:
Thanks updated